interfaces/builtin/i2c.go

```diff
@@ -87,17 +88,23 @@ func (iface *I2cInterface) PermanentSlotSnippet(slot *interfaces.Slot, securityS
 	return nil, nil
 }
 
+func (iface *I2cInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.Plug, slot *interfaces.Slot) error {
+	path, pathOk := slot.Attrs["path"].(string)
+	if !pathOk {
+		return nil
+	}
+
+	cleanedPath := filepath.Clean(path)
+	return spec.AddSnippet(fmt.Sprintf("%s rw,\n", cleanedPath))
```

Comment:
Indeed. I've spotted a couple of those in other tests too and fixed them in this PR. Thanks.